### ISSUE TITLE
Steam Audio is not initialized when using Real Time Audio in Unreal Engine

### ISSUE THREAD

**Issue by mastercoms** [ValveSoftware/steam-audio#206]
```cppp
IPLSimulationSettings FSteamAudioManager::GetRealTimeSettings(IPLSimulationFlags Flags)
{
    check(bSettingsLoaded); // <--- fails from OnInitSource call (callback from audio mixer)
```

**Comment by mastercoms** [comment]
Real Time audio can be enabled in the level editor viewport to listen to sounds without a PIE session.

**Comment by lakulish** [comment]
We have just released Steam Audio v4.4.1 (https://github.com/ValveSoftware/steam-audio/releases/tag/v4.4.1), which contains a fix for this issue.
